plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -1480,10 +1480,46 @@ function woocommerce_result_count() {
 		if ( ! wc_get_loop_prop( 'is_paginated' ) || ! woocommerce_products_will_display() ) {
 			return;
 		}
-		$args = array(
-			'total'    => wc_get_loop_prop( 'total' ),
-			'per_page' => wc_get_loop_prop( 'per_page' ),
-			'current'  => wc_get_loop_prop( 'current_page' ),
+
+		/**
+		 * Filters the default orderby option.
+		 *
+		 * @since 1.6.4
+		 *
+		 * @param string  $default_orderby The default orderby option.
+		 */
+		$default_orderby = apply_filters( 'woocommerce_default_catalog_orderby', get_option( 'woocommerce_default_catalog_orderby', '' ) );
```

Comment:
There is no guarantee this will be a `string` once all filters are run. Please can we add a type check when using this value?

Reply:
@tjcafferkey Thanks for catching that!
The `$default_orderby` variable is used to set the `$orderby` value, so I decided to add this piece of code as the final statement to set it:
```php
$orderby = is_string( $orderby ) ? $orderby : '';
```

Reply:
Thanks!